src/Wrapper.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Rubix\ML;
+
+/**
+ * Wrapper
+ *
+ * @category    Machine Learning
+ * @package     Rubix/ML
+ * @author      Andrew DalPino
+ */
+interface Wrapper extends Estimator
```

Comment:
I'm not 100% sold on the name of this interface ... on one hand, if we only ever have one type of "wrapper" object then this name would be perfect, but on the other hand, if we implement different types of wrappers then ModelWrapper or EstimatorWrapper or something like that would be more appropriate. Let me think about this one for a little bit. I'm leaning toward keeping it the way it is unless I can find a compelling use case for another type of wrapper.

Reply:
Done 👍